charts/access/email/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
```

Comment:
Is this comment wrong now that we are pinning the chart versions to the Teleport version?

Reply:
That's true. Fixed in https://github.com/gravitational/teleport-plugins/pull/532/commits/108aa3fc57edaa2ac17b91e5f87308b41a40e569